docs/spec/manifest-v2-2.md

```diff
@@ -24,7 +24,6 @@ an ID for the image.
 The following media types are used by the manifest formats described here, and
 the resources they reference:
 
-- `application/vnd.docker.distribution.manifest.v1+json`: schema1 (existing manifest format)
```

Comment:
Awesome, we also need to remove https://github.com/distribution/distribution/blob/ff5f31a7a57a65a236b9f241ff61ec3d8f024f5a/docs/spec/manifest-v2-2.md?plain=1#L63